spring-boot-project/spring-boot-docs/src/docs/antora/modules/reference/pages/features/logging.adoc

```diff
@@ -450,6 +450,35 @@ Spring Boot supports structured logging and has support for the following JSON f
 
 To enable structured logging, set the property configprop:logging.structured.format.console[] (for console output) or configprop:logging.structured.format.file[] (for file output) to the id of the format you want to use.
 
+If you are using xref:#features.logging.custom-log-configuration[Custom Log Configuration], update your configuration to respect `CONSOLE_LOG_STRUCTURED_FORMAT` and `FILE_LOG_STRUCTURED_FORMAT` system properties.
+Take `CONSOLE_LOG_STRUCTURED_FORMAT` for example:
+[tabs]
+======
+Logback::
++
+[source,xml]
+----
+<!-- replace your encoder with StructuredLogEncoder -->
+<encoder class="org.springframework.boot.logging.logback.StructuredLogEncoder">
+	<format>${CONSOLE_LOG_STRUCTURED_FORMAT}</format>
+	<charset>${CONSOLE_LOG_CHARSET}</charset>
+</encoder>
+----
+Log4j2::
++
+[source,xml]
+----
+<!-- wrap your PatternLayout with <Select> and <DefaultArbiter> -->
+<Select>
```

Comment:
I think we can skip the whole select and arbiter stuff here. The user wants to have structured logging, no need for conditionals. The logback section isn't conditional, too.

Reply:
FYI, my actual case requires conditional.

Reply:
I agree with @mhalbritter. We shouldn't over-complicate the documentation by showing the use of conditionals if they aren't always needed.

Reply:
OK, I added links to default configurations for fine-grained control.